platform/src/main/java/org/stellar/anchor/platform/utils/SecurityUtil.java

```diff
@@ -0,0 +1,83 @@
+package org.stellar.anchor.platform.utils;
+
+import com.google.common.io.BaseEncoding;
+import io.jsonwebtoken.SignatureAlgorithm;
+import java.security.InvalidKeyException;
+import java.security.KeyFactory;
+import java.security.NoSuchAlgorithmException;
+import java.security.PublicKey;
+import java.security.Signature;
+import java.security.SignatureException;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.X509EncodedKeySpec;
+import java.util.Base64;
+import org.apache.commons.lang3.StringUtils;
+
+public class SecurityUtil {
```

Comment:
Maybe rename to RSAUtil?
Pretty sure only RSA keys have the format that is used in this class